openstef/tasks/optimize_hyperparameters.py

```diff
@@ -44,6 +44,10 @@ def optimize_hyperparameters_task(
         context: Task context
 
     """
+    if pj.pipelines_to_run == PipelineType.FORECAST:
```

Comment:
See earlier comment about implementing the pipelines_to_run as a list. In that case the check here would be to check if the pipelines_to_run contains a hyper_parameters type.